k9mail/src/main/java/com/fsck/k9/Account.java

```diff
@@ -953,8 +953,8 @@ public void setLocalStorageProviderId(String id) {
                 switchLocalStorage(id);
                 successful = true;
             } catch (MessagingException e) {
-                Log.e(K9.LOG_TAG, "Switching local storage provider from " +
-                      mLocalStorageProviderId + " to " + id + " failed.", e);
+                Timber.e("Switching local storage provider from " +
```

Comment:
We should probably switch away from doing string concatenations in log messages. Timber supports format strings. So this could be:

```java
Timber.e(e, "Switching local storage provider from %s to %s failed", mLocalStorageProviderId, id);
```